src/main/java/org/jason/fgcontrol/flight/position/KnownPositions.java

```diff
@@ -106,4 +106,10 @@ private KnownPositions() {}
     public final static WaypointPosition RATLAM_CITY = new WaypointPosition(23.33676, 75.00124, "Ratlam City");
     public final static WaypointPosition KOTA_CITY = new WaypointPosition(25.17353, 75.76456, "Kota City");
     public final static WaypointPosition DELHI_CITY = new WaypointPosition(28.64428, 76.92842, "Delhi City");
+
+    //North California Tour
+    public final static WaypointPosition SFO_CITY = new WaypointPosition(37.8199328, -122.4808354, "SFO City");
```

Comment:
This isn't explicitly documented but 5 decimals places of precision for lat/lon is suitable for our purposes.

Reply:
Updated lat/lon to 5 decimals